apps/web/src/contexts/globalLayout/EditGalleryNavbar/EditGalleryNavbar.tsx

```diff
@@ -64,6 +69,10 @@ export function EditGalleryNavbar({
   const isMobile = useIsMobileOrMobileLargeWindowWidth();
 
   const handleAllGalleriesClick = useGuardEditorUnsavedChanges(() => {
+    if (doneAction === 'saved') {
+      onDone('');
+    }
+
```

Comment:
can we leave a comment here explaining that this auto saves the user's gallery? if that's what it does

Reply:
+1

@tbezman when this hook is instantiated will it lock in the value of `doneAction`? in other words can we be confident that the callback will be up-to-date when `doneAction` changes

Reply:
nah it's always up to date

Reply:
Got it! Added the comment